site/assignments/starbucks/starbucks.rst

```diff
@@ -77,6 +77,22 @@ Part 2 --- Read Over Loading Function
 Part 3 --- Degrees to Decimal
 =============================
 
+Humans like to express `latitudes <http://en.wikipedia.org/wiki/Latitude>`_ and
+`longitudes <http://en.wikipedia.org/wiki/Longitude>`_ in degrees, minutes, and seconds. You *could* work with those
+units in Python, but your computations (and, hence, code) will be much cleaner and easier to follow if you convert the
+data into the single unit "degrees", using decimals of a degree to represent "arcminutes" (1/60th of a degree) and
+"arcseconds" (1/60th of a arcminute).
+
+Complete the function ``convert_degrees_to_decimal`` such that it converts the provided degrees, arcminutes, and
+arcseconds to decimal. The parameters are a latitude/longitude in degrees, arcminutes, and arcseonds. The function
+should return the same latitude/longitude as a single value in decimal degrees (a single value of type ``float``).
+
+You are likely not familiar with the conversion, but fortunately
+`Wikipedia is <http://en.wikipedia.org/wiki/Arcminute>`_.
```

Comment:
Maybe better is something like: _if you're not familiar with the conversion, check \<wiki\>_